dev/com.ibm.ws.install.utility/resources/com/ibm/ws/install/utility/internal/resources/InstallUtilityMessages.nlsprops

```diff
@@ -0,0 +1,230 @@
+
+ERROR_NO_CONNECTION=CWWKF1431E: Cannot establish a connection to the configured repository server.
+ERROR_NO_CONNECTION.explanation=The connection to the repository server cannot be established.
+ERROR_NO_CONNECTION.useraction=See the problem determination information on the WebSphere Application Server Support Web page: http://www.ibm.com/software/webservers/appserv/was/support.
+
+ERROR_MORE_THAN_0_OR_1_ARGUMENTS=CWWKF1432E: {0} requires at most 1 argument, but {1} were provided.
+ERROR_MORE_THAN_0_OR_1_ARGUMENTS.explanation=The wrong number of arguments were used to invoke the action.
+ERROR_MORE_THAN_0_OR_1_ARGUMENTS.useraction=Rerun the command with the correct arguments.
+
+ERROR_TOOL_INCORRECT_PROXY_CREDENTIALS=CWWK1433E: The credentials for the configured proxy server cannot be authenticated. Ensure that valid credentials for the proxy server are set in the following configuration file: {0}
+ERROR_TOOL_INCORRECT_PROXY_CREDENTIALS.explanation=The configured proxy server requires authentication, but the credentials cannot be authenticated.
+ERROR_TOOL_INCORRECT_PROXY_CREDENTIALS.useraction=Specify valid proxy server credentials in the configuration file, and run the command again.
+
+ERROR_UNABLE_TO_FIND_SERVER_XML=CWWK1434E: Unable to locate the server.xml file in directory {0}  Ensure that the file exists and is accessible.
+ERROR_UNABLE_TO_FIND_SERVER_XML.explanation=The server.xml file for the specified server cannot be located in the server directory.
+ERROR_UNABLE_TO_FIND_SERVER_XML.useraction=Ensure that the file exists and is accessible, then rerun the command.
+
+ERROR_UNABLE_TO_ACCESS_SERVER_XML=CWWK1435E: Unable to access the file {0} Ensure that the user account can access and read the file.
+ERROR_UNABLE_TO_ACCESS_SERVER_XML.explanation=The user account does not have the required permissions to read the server.xml file.
+ERROR_UNABLE_TO_ACCESS_SERVER_XML.useraction=Ensure that the user account can access and read the file and rerun the command.
+
+MSG_REPO_FILE_NOT_FOUND=No user configuration detected. The IBM WebSphere Liberty Repository is the default assets repository.
+MSG_REPO_FILE_NOT_FOUND_EXPLANATION=To customize installUtility settings, create a repositories.properties file at the following location: {0}
+MSG_REPO_FILE_NOT_FOUND_ACTION=Use the following sample as a template to create your own repositories.properties file. Uncomment the lines marked by a single # character, and replace the values with your own customized values.
+
+ERROR_UNINSTALL_FEATURES_FAIL=CWWK1436E: One or more features cannot be uninstalled: {0}
+ERROR_UNINSTALL_FEATURES_FAIL.explanation=Internal error detected while processing metadata. 
+ERROR_UNINSTALL_FEATURES_FAIL.useraction=See the problem determination information on the WebSphere Application Server Support Web page: http://www.ibm.com/software/webservers/appserv/was/support.
```

Comment:
Please ask WAS support for the correct URL to use in messages. http://www.ibm.com/software/webservers/appserv/was/support is redirected and might no longer be the correct URL to use in messages.